src/main/java/com/blog/dao/impl/TagDaoRestClientImpl.java

```diff
@@ -46,4 +61,43 @@ public TagDaoRestClientImpl(RestTemplate restTemplate, HttpHeaders headers) {
         );
         return tags.getBody();
     }
-}
+
+    public void addTag(Tag tag) {
```

Comment:
Where is exception handling? What if service unavailable? At least response status code should be validated

Reply:
RestTemplate throws errors on all codes 4 ** and 5 **, so exception handling will take place in the controllers.